### PR TITLE
 Document assertEq for arrays and Std Logs (Forge Std)

### LINKED ISSUE #372: Document assertEq for arrays and Std Logs (Forge Std)
### assertEq

It is now possible to assert equality of `uint` and `int` arrays with `assertEq`. See the code [here](https://github.com/foundry-rs/forge-std/blob/e0393ac558fa62731e7890677c5f0559a06e39c2/src/Test.sol#L236-L266).

### Std Logs

New logs are available. See [Std Logs](https://github.com/foundry-rs/forge-std/blob/e0393ac558fa62731e7890677c5f0559a06e39c2/src/Test.sol#L16-L23). Those should have a dedicated section like Std Cheats, Std Math, etc.

### PR DESCRIPTION
**Pull request by 0xvv**
Resolves #372

### CODE REVIEW

File: src/reference/forge-std/std-logs.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+event log_array(int256[] val);
+event log_named_array(string key, uint256[] val);
+event log_named_array(string key, int256[] val);
+```
+
+## Usage
+
+This section provides usage examples.
+
+### log\_array\<type\>
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:20]: No type here
```suggestion
### log\_array
```

File: src/reference/forge-std/std-logs.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+```solidity
+event log_array_<type>(<type>[]);
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:23]: Same as above
```suggestion
event log_array(<type>[] val);
```

File: src/reference/forge-std/std-logs.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Where `<type>` can be `int256` or `uint256`.
+
+#### Example
+
+```solidity
+uint256[] data = [10, 20, 30, 40, 50]; 
+emit log_array(data);
+```
+
+### log\_named\_array\<type\>
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:35]: Same as above
```suggestion
### log\_named\_array
```

File: src/reference/forge-std/std-logs.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+```solidity
+event log_named_array_<type>(string key, <type>[] val);
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:38]: Same as above
```suggestion
event log_named_array(string key, <type>[] val);
```

File: src/reference/forge-std/std-logs.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+Where `<type>` can be `int256` or `uint256`.
+
+#### Example
+
+```solidity
+int256[] data = [10, 20, 30, 40, 50]; 
+emit log_named_array("Data :", data);
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:47]: Small nit - Forge will add a colon here
```suggestion
emit log_named_array("Data", data);
```

File: src/reference/forge-std/std-logs.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+```solidity
+event log_array_<type>(<type>[]);
+```
+
+Where `<type>` can be `int256` or `uint256`.
+
+#### Example
+
+```solidity
+uint256[] data = [10, 20, 30, 40, 50];
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:31]: Specify memory location
```suggestion
uint256[] memory data = [10, 20, 30, 40, 50]; 
```

File: src/reference/forge-std/std-logs.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+```solidity
+event log_named_array_<type>(string key, <type>[] val);
+```
+
+Where `<type>` can be `int256` or `uint256`.
+
+#### Example
+
+```solidity
+int256[] data = [10, 20, 30, 40, 50];
```
Comment (ZeroEkkusu) [inline src/reference/forge-std/std-logs.md:46]: Same as above

```suggestion
int256[] memory data = [10, 20, 30, 40, 50]; 
```

**Review by onbjerg — APPROVED** [review]
Thanks :heart:

**Pull request merged**